package.json

```diff
@@ -46,7 +46,7 @@
     "test": "standard && nyc tape test/*-test.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
-    "prebuild": "prebuildify --napi --strip",
+    "prebuild": "prebuildify -t $(node -v) -t electron@3.0.0 --napi --strip",
```

Comment:
Not sure if `$(node -v)` is kosher for windows.

Reply:
It is not, unless we switch to travis for windows ;)

Reply:
Perhaps move this logic to `travis.yml` and `appveyor.yml`

Reply:
Hmm, but we do `npm run prebuild` from those files. We could hardcode the node version number for now and pull this back in after we switch to travis for windows?

Reply:
We could do `npm run prebuild -- -t ..`. A bit ugly. I'm fine with hardcoding 👍

Reply:
Another thing, it seems that appveyor tries to do `npm run prebuild` https://ci.appveyor.com/project/peakji/leveldown/builds/21045244/job/rvmujh6v9kqrjdfd#L2339 , which it shouldn't?